USAGE.md

```diff
@@ -0,0 +1,114 @@
+# USAGE
+
+This document describes how you can use AMT without building it.
+
+## Using AMT
+
+To use AMT without building we recommend the pre-build docker images on [github](https://github.com/MinBZK/amt/pkgs/container/amt).
+
+You can deploy AMT to kubernetes or run the container locally using docker compose.
+
+-   Example [kubernetes](https://github.com/MinBZK/ai-validation-infra/tree/main/apps/amt)
+-   Example [docker compose](./compose.yml)
+
+To run amt locally create a compose.yml file and install [docker desktop](https://www.docker.com/products/docker-desktop/). Once you have install docker you can run the following command in the directory where you created the compose.yml
```

Comment:
Why create a new compose.yml if one can also use the existing one in the root?

Reply:
Or is this purely because we mount a volume and this is dependent on the webpack build?

Reply:
In this case, can't we comment in the yaml to remove this line for these developers?

Reply:
i want to make it easier for users (since they may not be developers). There are significant changes in the yaml compared to the one in our root